db_diff/tskdbdiff.py

```diff
@@ -285,10 +286,16 @@ def _dump_output_db_bb(db_file, bb_dump_file, isMultiUser, pgSettings, id_obj_pa
                             elif attr["value_type"] == 4:
                                 attr_value_as_string = "bytes"                        
                             elif attr["value_type"] == 5:
-                                attr_value_as_string = str(attr["value_int64"])                        
+                                attr_value_as_string = str(attr["value_int64"])
+                            elif attr["value_type"] == 6:
+                                attr_value_as_string = str(attr["value_text"])
                             if attr["display_name"] == "Associated Artifact":
                                 attr_value_as_string = TskDbDiff._get_associated_artifact_type(attribute_cursor, attr_value_as_string, isMultiUser)
                             patrn = re.compile("[\n\0\a\b\r\f]")
+                            if attr_value_as_string == None:
+                                print(f'{attr["value_type"]}')
+                                print(f'{attr["display_name"]}')
```

Comment:
I think these are good checks to have, but could it be a little more descriptive of the problem in case we need to debug this later (i.e. `print(f'Could not determine attribute value for value type: {attr["value_type"]}, display name: {attr["display_name"]}')` )?